recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,6 @@
-c_stdlib_version:          # [osx and x86_64]
-- '10.15'                  # [osx and x86_64]
 github_actions_labels:
 - cirun-openstack-cpu-large
+c_compiler_version:    # [linux]
```

Comment:
@h-vetinari Any obvious things that I'm doing wrong here?

Note, I added `skip: true` to `meta.yaml` to save CI resources.

Reply:
> Any obvious things that I'm doing wrong here?

Global pinning has two pins (if you're in a cuda-recipe): one for None and one for 12.6. To be able to override the key, you need to set values of the same length